.travis.yml

```diff
@@ -14,10 +14,12 @@ cache:
 
 before_install:
   - phpenv config-rm xdebug.ini
+  - wget http://cs.sensiolabs.org/download/php-cs-fixer-v2.phar --output-document php-cs-fixer.phar
```

Comment:
I recommend to download the PHAR from GitHub (https://github.com/FriendsOfPHP/PHP-CS-Fixer/releases) because:

* It is available trough HTTPS
* You can stick to a specific version, it makes the tests more reliable